katello/helper.rb

```diff
@@ -28,28 +40,63 @@ def disable_system_check_option?
     end
 
     def fail_with_message(message, opt_parser=nil)
-      STDOUT.puts message
+      STDOUT.puts message.red
       puts opt_parser if opt_parser
       exit(false)
     end
 
-    def run_cmd(command, exit_codes=[0], message=nil)
+    def fail_if_file_not_found(files)
```

Comment:
How close are we trying to get with the cherry pick? Asking because it looks good for the most part but this `fail_if_file_not_found` does not look familiar to me, so it probably belongs to another PR and not the one you've cherry picked

Reply:
That is from this commit here https://github.com/theforeman/foreman-packaging/commit/ea534f6227781721bfcc9922b72c0d77ee1e9b69

@akofink are the custom cert options for k-c-h included in 3.5?

Reply:
Weren't they added in https://github.com/theforeman/foreman-packaging/commit/ea534f6227781721bfcc9922b72c0d77ee1e9b69, or are there other code changes required?